src/tree/box.js

```diff
@@ -21,5 +32,16 @@ export default function box(divBox, o, appendNodes, cardStyling, reRender) {
         reRender(node.data.id);
```

Comment:
close tooltips on click as well, else when the tree re-renders you'll never get a mouseout